assets/js/myparcel.js

```diff
@@ -5,49 +5,47 @@ MyParcel = {
      * Initialize the MyParcel checkout.
      *
      */
-    data: {},
+    data:            {},
     currentLocation: {},
 
-    DELIVERY_MORNING: 'morning',
-    DELIVERY_NORMAL: 'standard',
-    DELIVERY_EVENING: 'avond',
-    DELIVERY_SIGNATURE: 0,
+    DELIVERY_MORNING:        'morning',
+    DELIVERY_NORMAL:         'standard',
+    DELIVERY_EVENING:        'avond',
+    DELIVERY_SIGNATURE:      0,
     DELIVERY_ONLY_RECIPIENT: 0,
 
     SPLIT_STREET_REGEX: /(.*?)\s?(\d{1,4})[/\s\-]{0,2}([a-zA-Z]{1}\d{1,3}|-\d{1,4}|\d{2}\w{1,2}|[a-zA-Z]{1}[a-zA-Z\s]{0,3})?$/g,
 
-    init: function()
-    {
-        this.data = myParcelConfig;
```

Comment:
Wanneer je ` this.data = myParcelConfig;` weghaalt zal alle instellingen die je heb ingevuld in de backend niet meer gevonden kunnen worden in dit bestand.